src/main/java/Bellroy/Bellroy.java

```diff
@@ -0,0 +1,46 @@
+package Bellroy;
+
+import java.io.IOException;
+
+/**
+ * Represents an instance of the Bellroy Chatbot.
+ */
+
+public class Bellroy {
```

Comment:
Minor coding standard adherence that can be done here by removing line 8:
"No blank line between the documentation block and the method/class."